app/models/picture.rb

```diff
@@ -0,0 +1,10 @@
+class Picture < ActiveRecord::Base
+
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  has_attached_file :image, styles: {medium: "300x300>", thumb: "100x100>"},
+                    default_url: "/images/:style/missing.png"
+  validates_attachment_content_type :image, content_type: %r(Aimage/.*\Z)
```

Comment:
Style/PercentLiteralDelimiters: `%r`-literals should be delimited by `{` and `}`. (https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)